yepun/cnpg/cnpg.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -xe
+
+# Install cloudnativePG on cluster (chart 0.17.0 installs 1.19.0 cnpg)
+helm repo add cnpg https://cloudnative-pg.github.io/charts
+helm upgrade --install cnpg \
+  --version="0.17.0" \
+  --namespace cnpg-system \
+  --create-namespace \
+  cnpg/cloudnative-pg \
+  --atomic
+
+# create namespace for deployment
+kubectl create namespace cloudnativepg
+
+# Secrets - app user - postgres user - AWS account for backups
+cat << END | kubectl apply -f -
+---
+apiVersion: v1
+data:
+  password: $(echo -n "${SUPERUSER_PASSWORD}" | base64)
+  username: $(echo -n "postgres" | base64)
+kind: Secret
+metadata:
+  name: cnpg-cluster-superuser
+  namespace: cloudnativepg
+type: kubernetes.io/basic-auth
+END
+
+# Deployment FIRST TIME ONLY
+cat > deploy.yaml << END
```

Comment:
I realize we do this all over the place, but as a general comment, inlining yaml inside a bash script does prevent it from being linted. This is something we should discuss and figure out what we want our standard to be.